Themes/default/PersonalMessage.template.php

```diff
@@ -463,15 +464,15 @@ function template_single_pm($message)
 				</ul>
 			</div><!-- .poster -->
 			<div class="postarea">
-				<div class="flow_hidden">
-					<div class="keyinfo">
-						<h5 id="subject_', $message['id'], '">
-							', $message['subject'], '
-						</h5>';
+				<div class="keyinfo">
+					<div id="subject_', $message['id'], '" class="subject_title">
+						', $message['subject'], '
+					</div>
+					<div class="postinfo">';
```

Comment:
Unfortunately, the lines inside this div still aren't being laid out as desired. I suggest adding a rule like this to index.css:
```
#personal_messages .keyinfo .postinfo span {
	display: block;
	width: 100%;
}
```

Reply:
I changed the direction instead, should do the same 6738486

Reply:
Even better. 🙂